src/index.html

```diff
@@ -12,7 +12,65 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="stars">
+    <div class="stars stars--0">
+      <ul class="stars__list">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to simplify reading and add some 'air' to the code structure. There should be an empty line before and after each `div` block containing the `stars__list`.